README.md

```diff
@@ -410,6 +402,66 @@ for this interface (it would be used in generated code to provide proper enum va
 The impl-block with trait implementation also contains the `#[messages]` attribute,
 but only one - the one with info about the trait being implemented.
 
+## Macro attributes
+
+`Sylvia` work with multiple attributes. I will explain here how and when to use which of them.
+
+```rust
+#[contract(module=contract_module::inner_module)]
+impl Interface for MyContract {
+...
+}
+```
+
+`module` is meant to be used when implementing interface on the contract. It's purpose
+is to inform `sylvia` where is the contract defined. If the contract is implemented in the same
+scope this attribute can and should be omitted.
+
+```rust
+#[entry_point]
```

Comment:
Nice to add a line about `entry_point` here as well (it is above, maybe just copy)